_posts/2023-11-09-platform-protections.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+date: November 9th 2023
+title: "New platform protections against malicious activity"
+---
+
+# New platform protections against malicious activity!
```

Comment:
You don't need this because Jekyll will automatically print the `title` from above as the header on the page